### PR TITLE
BaSP-MR-46: Trainer resource with GET, POST, PUT and DELETE endpoints

### PR DESCRIPTION
**Pull request by MauroCaffesse**
When accessing the "Trainer" page, the complete list of active trainers should be displayed, if any. An "Edit" button has been implemented, which, when clicked, should open a modal form with the preloaded trainer data. Additionally, a "Delete" button has been implemented, which, upon clicking, displays a modal to confirm or cancel the deletion. Below the table, a button to add a trainer has been created. When pressed, the same modal with the form should appear, but with empty fields to be filled out.

### CODE REVIEW

**Review by victoriassol — APPROVED** [review]
Perfection! I take off my hat 🤌🏼👒

File: src/Components/Trainers/index.jsx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    } catch (error) {
+      setResponseMessage(`Error fetching trainers: ${error.message}`);
+      setShowResponseModal(true);
+    }
+  };
+  useEffect(() => {
+    getTrainers();
+  }, []);
+
+  /*   const getTrainer = async (id) => {
```
Comment (JuanCruzMoreira) [inline src/Components/Trainers/index.jsx:29]: Delete commented lines please

File: src/Components/Trainers/index.jsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  };
+
+  return (
+    <section className={styles.container}>
+      <h2>Trainers</h2>
+      {activeTrainers.length > 0 ? (
+        <Table
+          data={activeTrainers}
+          deleteTrainer={deleteTrainer}
+          /* getTrainer={getTrainer} */
```
Comment (JuanCruzMoreira) [inline src/Components/Trainers/index.jsx:112]: Delete this line too

**Review by JuanCruzMoreira — CHANGES_REQUESTED** [review]
You must add error messages in create/edit form

**Review by JuanCruzMoreira — APPROVED** [review]

**Pull request merged**